pkg/apis/objectbucket.io/v1alpha1/objectbucketclaim_types.go

```diff
@@ -31,7 +31,7 @@ type ObjectBucketClaimSpec struct {
 	Versioned bool `json:"versioned"`
 	// AdditionalConfig gives non-AWS S3 providers a location to set
 	// proprietary config values (tenant, namespace, etc)
-	AdditionalConfig map[string]interface{} `json:"additionalConfig"`
+	//AdditionalConfig map[string]interface{} `json:"additionalConfig"`
```

Comment:
What about naming this field a name consistent with what's used in a storage class or config map to hold extra values, eg `Parameters` or `Data` or ?